frc/1. Basics/1d. Vendor Dependencies.md

```diff
@@ -0,0 +1,14 @@
+# Vendor Dependencies
+
+When we use hardware from a third-party source, we need to place what are called "vendor dependency" files into our robot program. 
+
+A prime example of such hardware would be a motor controller (see [Motors]() for more detail). Say we are using a SPARK MAX controller from REV Robotics--in order for the controller to actually function, we need to get the vendor dependency (a `.json` file) from REV Robotics' [website](http://www.revrobotics.com/sparkmax-software/).
+
+The following steps for managing vendor dependencies can be found on that site:
+
+<img src="img/REVVendorCheck.jpg">
+
+([Offline Library WPI Guide](https://docs.wpilib.org/en/latest/docs/software/wpilib-overview/3rd-party-libraries.html))
+
+
+Similar steps can be followed with other devices, such as the Talon SRX motor controller and navX-MXP gyroscope. What's most important is that we place these files into a folder called `vendordeps`, and put this folder on the **main** file level of the robot program (not in any subfolders like we had with the robot code files).
```

Comment:
There's a better way to do vendor dependencies, which is to right click build.gradle and go to Install Vendor Dependency (Online), and then you paste in a link. This also allows you to check for updates on your vendor dependencies.